sql/core/src/test/scala/org/apache/spark/sql/SQLQueryTestSuite.scala

```diff
@@ -51,6 +51,11 @@ import org.apache.spark.sql.types.StructType
  *   SPARK_GENERATE_GOLDEN_FILES=1 build/sbt "sql/test-only *SQLQueryTestSuite"
  * }}}
  *
+ * For selective tests, run:
```

Comment:
Why not directly using something like 
> build/sbt "~sql/test-only *SQLQueryTestSuite -- -z inline-table.sql -z random.sql"

Reply:
oh, I didn't know `sbt` can do. I usually use `maven` and it seems `maven` cannot do that. Anyway, it's okay to use `sbt for this case. Thanks!